packages/near-membrane-base/src/intrinsics.ts

```diff
@@ -214,17 +214,14 @@ export function getFilteredGlobalOwnKeys(source: object): (string | symbol)[] {
     return result;
 }
 
-export function linkIntrinsics(
-    env: VirtualEnvironment,
-    globalObjectVirtualizationTarget: typeof globalThis
-) {
-    // remapping intrinsics that are realm agnostic
+export function linkIntrinsics(env: VirtualEnvironment, blueGlobalObject: typeof globalThis) {
+    // Remap intrinsics that are realm agnostic.
     for (let i = 0, { length } = ReflectiveIntrinsicObjectNames; i < length; i += 1) {
         const globalName = ReflectiveIntrinsicObjectNames[i];
-        const reflectiveValue = globalObjectVirtualizationTarget[globalName];
+        const reflectiveValue = blueGlobalObject[globalName];
```

Comment:
This was named `globalObjectVirtualizationTarget` because when we have multi-window, this object won't be the blue global object when linkIntrinsics is called for a new iframe.contentWindow. Making it a "blue" thing would be confusing 🤷

Reply:
Ok.

Reply:
Another place had used `blueGlobalObject` so I was aligning but I can go the other way with it.

Reply:
That's my bad, I should've updated that one when the `globalObjectVirtualizationTarget` naming was introduced